cocotb/share/makefiles/simulators/Makefile.verilator

```diff
@@ -15,11 +15,17 @@ clean::
 
 else
 
-CMD := verilator
+CMD_BIN := verilator
 
-ifeq ($(shell which $(CMD) 2>/dev/null),)
-# Verilator is not in PATH, lets start searching for it
-$(error Cannot find verilator.)
+ifdef VERILATOR_BIN_DIR
```

Comment:
Is there a specific need for this feature? We require most other simulators to be in PATH.

Reply:
Every other simulator makefile supports a `<SIM>_BIN_DIR` environment variable. I also find it useful for my testing setup.

Reply:
Good with me then, sorry for the noise!